club_fair.md

```diff
@@ -0,0 +1,393 @@
+<!-- # Philosophy
+
+Hacker Space
+
+## First interactions matter
+
+# Getting up and running
+
+## Starting a new hackEDU club
+
+### Overview
+
+### Checklist -->
+
+# Club fair/rush day
+
+**Outline:**
+
+- **[Overview](#overview)**
+  - [Be a fanatic peacock](#be-a-fanatic-peacock)
```

Comment:
If you don't think people think you're crazy, you're probably not being crazy enough.